crates/bitcoin-da/src/verifier.rs

```diff
@@ -307,6 +313,35 @@ impl DaVerifier for BitcoinVerifier {
     }
 }
 
+fn verified_blob_chunk<'a, T, I>(
+    tx: &T,
+    blobs_iter: &mut I,
+    wtxid: &[u8; 32],
+) -> Result<Option<&'a [u8]>, ValidationError>
+where
+    T: VerifyParsed,
+    I: Iterator<Item = &'a BlobWithSender>,
+{
+    if let Some(blob_hash) = tx.get_sig_verified_hash() {
```

Comment:
get sig verified hash will fail because blob sigs and pubkeys are 0

impl. a new method like get_hash_sig_unverified 😂